datafusion/core/src/physical_optimizer/enforce_sorting.rs

```diff
@@ -2140,70 +2140,11 @@ mod tests {
         Ok(())
     }
 
-    #[tokio::test]
-    async fn test_with_lost_ordering_unbounded() -> Result<()> {
```

Comment:
I couldn't see a reason to keep these as they effectively duplicate the above

Reply:
These tests makes sure that when sources are unbounded, we try to preserve existing ordering as much as possible. I will re-add corresponding tests that enforces this functionality. For this PR, I just ignore these tests.